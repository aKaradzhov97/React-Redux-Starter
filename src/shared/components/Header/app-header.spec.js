import React from 'react';
import AppHeader from './app-header';
import { render } from '../../../tests/test-utils';

describe('AppHeader', () => {
  describe('Rendering', () => {
    it('should render the component', () => {
      const { container } = render(<AppHeader />);

      expect(container.querySelector('body section')).toMatchSnapshot();
    });
  });
});
