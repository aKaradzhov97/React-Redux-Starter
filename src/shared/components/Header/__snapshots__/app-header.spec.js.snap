// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader Rendering should render the component 1`] = `null`;
